incubator/blockchain-client-openliberty/README.md

```diff
@@ -180,6 +132,34 @@ The metrics endpoint is secured with a userid and password enabled through the c
 
 In order to lock down the production image built via `appsody build` this file is deleted during the Docker build of your application production image.  (The same file would be deleted if you happened to create your own file at this location as well).
 
+## Stack development
```

Comment:
Is this section meant to be the "Champ" section about extending the stack?

Reply:
Yes - just a placeholder.